includes/configuration_managers/class-newspack-ads-configuration-manager.php

```diff
@@ -117,6 +117,15 @@ public function delete_ad_unit( $id ) {
 			$this->unconfigured_error();
 	}
 
+	/**
+	 * Check whether the current screen should show ads.
+	 *
+	 * @return bool Returns true if ads should be shown.
+	 */
+	public function should_show_ads() {
+		return $this->is_configured() ? newspack_should_show_ads() : false;
```

Comment:
I would suggest adding a check for the existence of the function, to avoid fatals if the Newspack plugin is updated before Newspack Ads.

Reply:
https://github.com/Automattic/newspack-plugin/pull/482/commits/f4943f07a0976380ca1a142c9aabd8f1c272b3c3